alembic/versions/1ab5aa1e3054_start_associating_comments_with_.py

```diff
@@ -0,0 +1,28 @@
+"""Start associating comments with registered users and use user information
+as a display name instead of arbitrary author names.
+
+Revision ID: 1ab5aa1e3054
+Revises: 2c00861f3ff1
+Create Date: 2015-09-01 18:05:35.598253
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '1ab5aa1e3054'
+down_revision = '2c00861f3ff1'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+def upgrade():
+    op.drop_column('user_comments', 'author_name')
```

Comment:
We're going to lose all existing author names, but that's okay :) I don't think that'll have a large impact for our current users. @arahuja